kits/sre-command-center/prompts/flow-3-master-responder-the-fixer_instructor-llmnode-1_user_1.md

```
@@ -0,0 +1,2 @@
+Analyze the following production alert and provide structured triage classification:
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)